lib/plugins/babelRuntimePathResolver.js

```diff
@@ -2,11 +2,12 @@
 // in OBT's node_modules directory and not the module's
 'use strict';
 
-const path = require('path');
 const falafel = require('falafel');
 
 const prefix = /babel-runtime/;
-const rootDirectory = path.resolve(__dirname, '../../node_modules');
+
+// HACK: Can't require.resolve babel-runtime so use babel-core instead
+const rootDirectory = require.resolve('babel-core').replace(/(.*node_modules).*/, '$1');
```

Comment:
I don't know … :speak_no_evil:  … although that would be broken with the original implementation wouldn't it?